nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/PGPUtil.java

```diff
@@ -46,9 +46,8 @@
     public static final int BUFFER_SIZE = 65536;
     public static final int BLOCK_SIZE = 4096;
 
-    public static void encrypt(InputStream in, OutputStream out, String algorithm, String provider, int cipher, String filename,
-                               PGPKeyEncryptionMethodGenerator encryptionMethodGenerator) throws IOException, PGPException {
-
+    public static void encrypt(InputStream in, OutputStream out, String algorithm, String provider, int cipher, String filename, PGPKeyEncryptionMethodGenerator encryptionMethodGenerator) throws
+            IOException, PGPException {
         final boolean isArmored = EncryptContent.isPGPArmoredAlgorithm(algorithm);
```

Comment:
@trkurc In this diff, whitespace is the only change; @olegz was referring to his comment on PR 163 where he suggested a null check for the `algorithm` parameter and I made the change there. I will pull the change here as well.